data/org.freedesktop.impl.portal.ScreenCast.xml

```diff
@@ -81,6 +81,37 @@
               Whether to allow selecting multiple sources. Default is no.
             </para></listitem>
           </varlistentry>
+          <varlistentry>
+            <term>restore_data as</term>
+            <listitem><para>
+              The data to restore from a previous session.
```

Comment:
This should probably contain a tip for what to pass here, e.g. the same array of strings that was received in a previous response signal.

Also, why is it an array? Does the order of elements matter? Do they have meaning or is it just implementation specific blobs?

Reply:
This is just the easiest way to deal with the permission store. I'm not entirely happy with how the permission store is used in this branch, but I just couldn't meaningfully use the "value" field of the permission store without overcomplicating the code. Let me go through it once again...

Reply:
Switch to a more sophisticated usage of the permission store. Data is now stored as:

```
Table      Object                               App                       Permissions                    Data
screencast 1be22dff-xxx-yyyy-zzzz-4ffdf1602084 com.obsproject.Studio     yes                            ['monitor::AUO:0x313d:0x00000000']

```